etc/eslint/rules/stdlib.js

```diff
@@ -346,8 +346,7 @@ rules[ 'stdlib/jsdoc-checkbox-character-style' ] = [
 	{
 		'checked': 'x',
 		'unchecked': ' '
-	}
-];
+	}];
```

Comment:
@headlessNode Why these changes? The existing code should be fine.

Reply:
@kgryte The rule is taking the new-line between closing brackets as a space. Which should be the correct behavior, no?

Same thing happens with `eol-open-bracket-spacing` rule.

We don't allow the following:
```
var arr = [
    {
       'a': 1
    }
];
```

It should be:
```
var arr = [{
    'a': 1
}];
```

Reply:
Ah! The first one should be allowed for `eol-open-bracket-spacing`. The only thing we want to disallow is when someone does 

```
foo = [ {
    ...
} ];
```

That is it. The following should be perfectly valid.

```
foo = [{
    ...
}];

foo = [
    {
        ...
    }
];

foo = [ {...} ];
```

Reply:
@kgryte Makes sense. Thanks!